src/reducers/auth.ts

```diff
@@ -0,0 +1,25 @@
+import * as Constants from 'constants/auth'
+
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+type ActionType = {
+  type: string
+  payload: any
```

Comment:
Can we define a specific type for `payload` as per the data we are passing to it?

Reply:
Yes we can, fixed in 4e0dc88f4b4f35d34dc3dd69f1e3d07a5bbecc48